DependencyInjection/EkinoNewRelicExtension.php

```diff
@@ -43,52 +46,65 @@ public function load(array $configs, ContainerBuilder $container): void
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
 
-        if (!$config['enabled']) {
-            $config['monolog']['enabled'] = false;
-            $interactor = BlackholeInteractor::class;
-        } elseif (isset($config['interactor'])) {
-            $interactor = $config['interactor'];
-        } else {
-            // Fallback to see if the extension is loaded or not
-            $interactor = \extension_loaded('newrelic') ? NewRelicInteractor::class : BlackholeInteractor::class;
-        }
+        $container->setAlias(NewRelicInteractorInterface::class, $this->getInteractorServiceId($config));
+        $container->setAlias(TransactionNamingStrategyInterface::class, $this->getTransactionNamingServiceId($config));
```

Comment:
you need to call `$container->setAlias(...)->setPublic(false)` to make the alias private as well. otherwise it will be public in sf 3.4. I would make everything private now.